### PR TITLE
Refactor datastore configuration.

### PR DESCRIPTION
**Pull request by hacktobeer**
The datastore module is unstable and we only use it to create indexes (which is done by a shell script calling gcloud in the module....).

This PR refactors the code to have the indexes created and destroyed by the shell scripts instead of the Terraform scripts. This way we have more control over it.

### CODE REVIEW

File: modules/turbinia/data/index-empty.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+indexes:
```
Comment (berggren) [inline modules/turbinia/data/index-empty.yaml:1]: where is this file being used?

**Review by berggren — CHANGES_REQUESTED** [review]

File: modules/turbinia/data/index-empty.yaml (hunk 1 above)
Comment (hacktobeer) [reply modules/turbinia/data/index-empty.yaml:1]: This is the empty file needed to do the cleanup of the indexes. You need to provide a yaml file, glcoud compares the content of the yaml file with the current configuration and removed the indexes.

Comment (hacktobeer) [reply modules/turbinia/data/index-empty.yaml:1]: It's used in the previous cleanup.sh.

**Review by berggren — APPROVED** [review]
LGTM

**Pull request merged**